recipes/dacap-clip/all/conanfile.py

```diff
@@ -75,6 +70,8 @@ def generate(self):
         toolchain.variables["CLIP_ENABLE_IMAGE"] = self.options.get_safe("with_image", False)
         if is_msvc(self):
             toolchain.cache_variables["CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS"] = bool(self.options.shared)
+        if Version(self.version) < "1.10":
```

Comment:
@perseoGI please do let me know if this is the correct approach for this case where the new version I'm adding supports CMake 4, but older ones do not. Should we justa ccept that they won't build with CMake 4 or still patch the old versions like this?